src/ResourceManager/Sql/Commands.Sql/Instance Failover Group/Cmdlet/SetAzureSqlInstanceFailoverGroup.cs

```diff
@@ -0,0 +1,173 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.Sql.InstanceFailoverGroup.Model;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Sql.InstanceFailoverGroup.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to create a new Azure Sql Database Instance Failover Group
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, "AzureRmSqlDatabaseInstanceFailoverGroup",
+        ConfirmImpact = ConfirmImpact.Medium)]
```

Comment:
@jugeorge 

- You can remove the `ConfirmImpact` property
- Same comment about having an `OutputType` attribute
- Same comment about implementing `SupportsShouldProcess`